reana_commons/openapi_specifications/reana_server.json

```diff
@@ -429,6 +429,26 @@
                     "slurmcern"
                   ]
                 },
+                "dask_cluster_default_number_of_workers": {
+                  "title": "Default number of workers for Dask clusters",
```

Comment:
For the server-side variables, please see notes about documentation lines in https://github.com/reanahub/reana/pull/821/files